cdap-docs/_common/_themes/cdap/static/cdap.css_t

```diff
@@ -301,13 +301,13 @@ p.admonition-title:after {
 }
 
 p.rubric {
-    background-color: #E0E0E0;
-    font-size: 120%;
-    font-weight: normal;
+    background-color: #EDEDED; 
     color: #111;
+    font-size: 110%; 
+    font-weight:normal;
```

Comment:
Yes, in the `basic.css`, which I've derived our CDAP style from, `p.rubric` is `font-weight: bold;`.